ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/management-api-gnmi.j2

```diff
@@ -5,14 +5,25 @@
 
 ### Management API GNMI Summary
 
-{%     if management_api_gnmi.enable_vrfs is defined %}
```

Comment:
We still need this check to avoid printing empty table (without values) when enable_vrfs is not defined.

Reply:
I printed the values further down now when vrf is not defined or if it's set to default

Reply:
Ok